opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -76,13 +76,11 @@ static COMMON_ATTRIBUTES: Lazy<[KeyValue; 4]> = Lazy::new(|| {
 async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
     let _ = init_tracer()?;
     let _ = init_metrics()?;
-    let _ = init_logs();
+    let logger_provider = init_logs().unwrap();
```

Comment:
```suggestion
    // Opentelemetry will not provide a global API to manage the logger provider. Application users must manage the lifecycle of the logger provider on their own. Dropping logger providers will disable log emitting.
    let logger_provider = init_logs().unwrap();
```

Suggest to add some comments for education purpose. Maybe even link this PR on why global log provider is not supported